src/Messaging/SAF.Messaging.Redis/ServiceCollectionExtensions.cs

```diff
@@ -47,7 +48,7 @@ internal static IServiceCollection AddRedisMessagingInfrastructure(this IService
                     var msgCfg = new RedisMessagingConfiguration(cfg);
                     var redisCfg = new RedisConfiguration { ConnectionString = msgCfg.ConnectionString };
                     return new Messaging(sp.GetRequiredService<ILogger<Messaging>>(),
-                        CreateRedisConnection(redisCfg),
+                        CreateRedisConnection(redisCfg, sp.GetRequiredService<ILogger<Messaging>>()),
```

Comment:
Must logging be a required service here? Shouldn't it be enough to use `GetService` instead? But of course a missing logging service must be handled then (hint: `NullLogger.Instance`)